env0/resource_environment.go

```diff
@@ -254,6 +254,12 @@ func resourceEnvironmentRead(ctx context.Context, d *schema.ResourceData, meta i
 	if err != nil {
 		return diag.Errorf("could not get environment: %v", err)
 	}
+
+	if environment.IsArchived {
```

Comment:
Can you please bring this behavior on the slack channel? it might happen to users a lot so I think we should discuss this first

Reply:
I will, but why does it need to happen a lot? its just when they deleted the environment which means we need to destroy this in state

Reply:
As discussed we are not going to handle `environment` and `project` resources because they aren't really deleted but "archived"